Update README for AWS templates (#39)
IBM/ibm-satellite-storage: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md

# AWS EFS CSI Driver

AWS EFS CSI driver implements the CSI specification for container orchestrators to manage the lifecycle of Amazon EFS file systems.

## Prerequisites

- Currently only *static provisioning* is supported. This means an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html) needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.

> ## AWS EBS CSI Driver parameters & how to retrieve them

Reviewer: ```suggestion
## AWS EFS CSI Driver parameters & how to retrieve them
```

> No parameters required for this template.

Reviewer: Are there any optional parameters that the user can specify? If so, we should list those here.
```suggestion
No parameters required for this template. 
```

Author: There is no optional parameters required for AWS EFS.

## Default storage classes

| Storage class name | File system | IOPs | Size range | Reclaim policy |
| --- | --- | --- | --- | --- |
| `sat-aws-file-gold` | NFS | NA | NA | Delete | 

## Creating the AWS EFS CSI Driver storage configuration

**Example `sat storage config create` command**

```
ibmcloud sat storage config create --name aws-efs-conf --template-name aws-efs-csi-driver --template-version 1.0.0
```

## Creating the storage assignment

**Example `sat storage assignment create` command**

```
ibmcloud sat storage assignment create --name install-efs --cluster-group <cluster-group> --configuration aws-efs-conf
```

## Verifying your AWS EFS CSI Driver storage configuration is assigned to your clusters

> 

Reviewer: ```suggestion
List the EFS driver pods in the `kube-system` namespace and verify that the status is `Running`.
```

```
$ oc get pods -n kube-system | grep efs    
efs-csi-node-4gkzx                      3/3     Running   0          2d22h
efs-csi-node-r8g5d                      3/3     Running   0          2d22h
efs-csi-node-td4wc                      3/3     Running   0          2d22h
```

> 

Reviewer: ```suggestion

List the EFS storage classes.

```

# AWS EFS CSI Driver

AWS EFS CSI driver implements the CSI specification for container orchestrators to manage the lifecycle of Amazon EFS file systems.

## Prerequisites

> - Currently only *static provisioning* is supported. This means an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html) needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.

Reviewer: ```suggestion
- Currently only static provisioning is supported. Before you can create a Satellite storage configuration, you must manually provision an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html). After you create your AWS EFS instance, you can create a Satellite storage configuration which references your EFS and can be mount your EFS inside a container.
```
